ui-manchette/src/stories/ManchetteSplit.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import '@osrd-project/ui-core/dist/theme.css';
+import '@osrd-project/ui-manchette/dist/theme.css';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { SAMPLE_WAYPOINTS } from './assets/sampleData';
+import Manchette from '../components/Manchette';
+import TrackOccupancyManchette from '../components/TrackOccupancyManchette';
```

Comment:
If you want to create a story with a trackOccupancyManchette, then you should probably move your story in the root folder `/storybook`, where the track-occupancy-diagrams stories are already. There  you will be able to import the trackOccupancyManchette easily ;)